test/modules/op/avg_pool2d.py

```diff
@@ -34,22 +34,28 @@ def get_example_inputs(self):
 
 
 @use_onert
+@skip_if(
+    Version(torch.__version__) >= Version("2.9.0.dev"),
+    reason="From torch 2.9.0.dev, dynamic shape requires more torch ir conversions related to symbolic runtime assertions",
```

Comment:
For details, see https://github.com/Samsung/TICO/issues/248#issuecomment-3149772872

Reply:
@mhs4670go I think there's both up and down sides. `Version(torch.__version__) >= Version("2.9.0.dev"),` let us know that `_assert_scalar` is produced from torch 2.9.0.dev, so we can target the operation's support with torch upgrade.

Reply:
Okay. Let's adapt as we go. Then, `export_produces_assert_scalar` is just implemented as a fallback?

Reply:
I forgot to remove it 😓 I updated the PR!